cmake/FindSDL2_image.cmake

```
@@ -36,53 +36,101 @@
 # module, but with modifications to recognize OS X frameworks and
 # additional Unix paths (FreeBSD, etc).
 
-if(NOT SDL2_IMAGE_INCLUDE_DIR AND SDL2IMAGE_INCLUDE_DIR)
-  set(SDL2_IMAGE_INCLUDE_DIR ${SDL2IMAGE_INCLUDE_DIR} CACHE PATH "directory cache entry initialized from old variable name")
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+  set(SDL2_ARCH_64 TRUE)
+  set(SDL2_PROCESSOR_ARCH "x64")
+else()
+  set(SDL2_ARCH_64 FALSE)
+  set(SDL2_PROCESSOR_ARCH "x86")
+endif(CMAKE_SIZEOF_VOID_P EQUAL 8)
+
+GET_SDL_EXT_DIR(SDL_EXT_DIR "mixer")
```

```suggestion
GET_SDL_EXT_DIR(SDL_EXT_DIR "image")
```

Hopefully this does the trick!

---

I changed it but this issue still gives me this:

file:///tmp/Spectacle.CRmpjV/Screenshot_20220920_180824.png

---

unfortunatley not. See comment below ;(